charts/argocd-apps/values-test.yaml

```diff
@@ -53,31 +53,11 @@ applications:
           secretKeyRef:
             name: publisher
             key: FACT_CHECK_PASSWORD
-      - name: FACT_CHECK_REPLY_TO_ADDRESS
```

Comment:
Thanks, this will fix those `CreateContainerConfigError` errors we're seeing for publisher:
```
couldn't find key FACT_CHECK_REPLY_TO_ADDRESS in Secret apps/publisher
```